--- dcd-0.0.2/configure.in~	Wed May  8 09:14:53 2002
+++ dcd-0.0.2/configure.in	Wed May  8 09:14:53 2002
@@ -17,7 +17,7 @@
   [ INCLUDEPTHREAD=1 ])
 
 dnl Checks for libraries.
-AC_CHECK_FUNC(gethostbyname, [] AC_CHECK_LIB(nsl, gethostbyname))
+AC_CHECK_FUNC(gethostbyname, [], AC_CHECK_LIB(nsl, gethostbyname))
 AC_CHECK_FUNC(connect)
 if test $ac_cv_func_connect = no; then
 	AC_CHECK_LIB(socket, connect)
@@ -26,7 +26,7 @@
 if test "$FREEBSD" = 1 -a "$INCLUDEPTHREAD"=1 ; then
   LDFLAGS="$LDFLAGS -pthread"
 elif test "$INCLUDEPTHREAD"=1 ; then
-  AC_CHECK_LIB(pthread,main,,INCLUDEPTHREAD=0)
+  AC_CHECK_LIB(pthread,main, [],INCLUDEPTHREAD=0)
   AC_DEFINE(HAVE_MITPTHREADS)
 else
 	echo "********************************"
